collections/_infographics/policies/emissions-pricing-world.md

```diff
@@ -13,4 +13,56 @@ data-orig:
+
+The area of the largest circle (China) corresponds to 12.4 Gt <glossary id="co2eq">CO<sub>2</sub>eq</glossary> and the size of the smallest circle (Switzerland) to 0.05 Gt CO<sub>2</sub>eq. Individual measures have been divided into emission trading systems (ETS) and carbon tax, depending on how they effectively work. **Approaches to carbon pricing have evolved in recent years** and some cannot be directly assigned to one of the categories described here. The authors acknowledge that a different categorisation may apply.
+
+### More details on individual countries
+{% comment %}
+{% include preview-box.html
+    title="European emission allowances"
+    slug="emisni-povolenky-ets"
+    text="For a more detailed explanation of how the EU Emissions Trading Scheme works, see our explainer."
+%}
+{% endcomment %}
+* Canadian measures include both components of the federal system (ETS and carbon tax) as well as provincial measures that charge for carbon emissions.
+* The RGGI (Regional Greenhouse Gas Initiative) was the first greenhouse gas pricing measure in the United States and now involves ten northeastern states.
+* The European Emissions Trading Scheme also includes Norway, Iceland and Liechtenstein. In addition to an across-the-board ETS, some EU countries have implemented a carbon tax.
+* Japan's system includes the Satiama Prefecture ETS and the Tokyo ETS.
+* China has eight pilot emission trading systems in operation so far, with a national ETS to be launched in 2020. The coverage is based on preliminary unofficial estimates from an announcement by the State Development and Reform Commission of the People's Republic of China. The national ETS will initially cover only the power sector, with other sectors covered by individual pilot emission trading systems.
+
+## The underlying study of the infographic
+
+We took the infographic from a [study](https://openknowledge.worldbank.org/handle/10986/31755) published in 2019 by the Global Climate Change Group of the World Bank Group. **The study maps 57 initiatives for charging emissions at the state and regional levels and describes which countries choose which measures and how they implement them.** The most important conclusions of the study:
+
+* Greenhouse gas pricing initiatives cover **20% of global emissions**.
+* The number of initiatives is growing rapidly, with 38 implemented in the last 10 years.
+* 51% of covered emissions are below $10 per tonne CO2eq, while **the price that would have led to meeting the Paris Climate Agreement targets is $40-$80** per tonne CO2eq, depending on location, which in effect meets only 5% of covered emissions. Sweden has introduced the highest unit price at $127 per tonne  CO<sub>2</sub>eq.
+* The study also looks at so-called implicit emissions pricing, which includes, for example, higher taxation of fossil fuels. Therefore, the total share of emissions covered by any legislation is still slightly higher than the 20% mentioned above.
```

Comment:
To match the Czech original, let's change "The study also looks at" to  "The study also newly looks at"

Reply:
Updated